### ISSUE TITLE
Segmentation fault (core dumped) while logging in

### ISSUE THREAD

**Issue by jedrzejowski** [xyzz/acquisition#452]
Hi, whenever i try to log into acquisition, i get this message:

```
adam $ ./acquisition 
qt5ct: using qt5ct plugin
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open sqlite3 database.
Przerwane (zrzut pamięci) # Segmentation fault (core dumped) # in polish
```

I have tied several times and i also cleared location `~/.local/share/acquisition`.

Result of `./acquisition --test`
```
qt5ct: using qt5ct plugin
********* Start testing of TestItem *********
Config: Using QtTest library 5.10.1, Qt 5.10.1 (x86_64-little_endian-lp64 shared (dynamic) release build; by GCC 7.3.0)
PASS   : TestItem::initTestCase()
PASS   : TestItem::Parse()
PASS   : TestItem::cleanupTestCase()
Totals: 3 passed, 0 failed, 0 skipped, 0 blacklisted, 0ms
********* Finished testing of TestItem *********
********* Start testing of TestShop *********
Config: Using QtTest library 5.10.1, Qt 5.10.1 (x86_64-little_endian-lp64 shared (dynamic) release build; by GCC 7.3.0)
PASS   : TestShop::initTestCase()
PASS   : TestShop::SocketedGemsNotLinked()
PASS   : TestShop::TemplatedShopGeneration()
PASS   : TestShop::cleanupTestCase()
Totals: 4 passed, 0 failed, 0 skipped, 0 blacklisted, 38ms
********* Finished testing of TestShop *********
********* Start testing of TestUtil *********
Config: Using QtTest library 5.10.1, Qt 5.10.1 (x86_64-little_endian-lp64 shared (dynamic) release build; by GCC 7.3.0)
PASS   : TestUtil::initTestCase()
PASS   : TestUtil::TestModMatcher()
PASS   : TestUtil::cleanupTestCase()
Totals: 3 passed, 0 failed, 0 skipped, 0 blacklisted, 0ms
********* Finished testing of TestUtil *********
********* Start testing of TestItemsManager *********
Config: Using QtTest library 5.10.1, Qt 5.10.1 (x86_64-little_endian-lp64 shared (dynamic) release build; by GCC 7.3.0)
PASS   : TestItemsManager::initTestCase()
PASS   : TestItemsManager::BuyoutForNewItem()
PASS   : TestItemsManager::BuyoutPropagation()
PASS   : TestItemsManager::UserSetBuyoutPropagation()
PASS   : TestItemsManager::MoveItemNoBoToBo()
PASS   : TestItemsManager::MoveItemBoToNoBo()
PASS   : TestItemsManager::MoveItemBoToBo()
PASS   : TestItemsManager::ItemHashMigration()
PASS   : TestItemsManager::cleanupTestCase()
Totals: 9 passed, 0 failed, 0 skipped, 0 blacklisted, 16ms
********* Finished testing of TestItemsManager *********
```

System: Arch Linux

**Comment by xyzz** [comment]
should be fixed by 8b219a65e000624b7d95eaa67b8324ae8d52769d

**Comment by jedrzejowski** [comment]
Ok, works now, ty
